app/src/main/java/com/kickstarter/ui/activities/ProjectActivity.kt

```diff
@@ -84,6 +84,16 @@ class ProjectActivity : BaseActivity<ProjectViewModel.ViewModel>() {
                     }
                 }
 
+        this.viewModel.outputs.setRewardButtonColor()
+                .compose(bindToLifecycle())
+                .observeOn(AndroidSchedulers.mainThread())
+                .subscribe { it?.let { native_back_this_project_button.setBackgroundColor(getColor(it)) } }
```

Comment:
![Screen Shot 2019-06-10 at 12 11 36 PM](https://user-images.githubusercontent.com/1289295/59209280-ed97dd80-8b78-11e9-94d2-9a2c012d44d0.png)

This code has a warning. The inner or outer `it` should be renamed.